src/UserService.Validation/Login/Interfaces/ILoginValidator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.UserService.Validation.Login.Interfaces
+{
+    [AutoInject]
+    public interface ILoginValidator : IValidator<string>
```

Comment:
why validator only for 'login'. make it validator for CreateCredentialsRequest, and validate that UserId exists

Reply:
done